### ISSUE TITLE
Monitors created with periphemu

### ISSUE THREAD

**Issue by magiczocker10** [MCJack123/craftos2#122]
**Describe the bug**
When creating a new monitor with periphemu, the monitor won't be listed in peripheral.getNames().

**To Reproduce**
1. periphemu.create("any id","monitor")
2. peripheral.getNames()

**Expected behavior**
Get a list **with** the created monitor.

**Screenshots**
![grafik](https://user-images.githubusercontent.com/59317952/86824195-1e08be80-c08e-11ea-96ee-bc8fe9c5b102.png)

**Environment (please complete the following information):**
 - OS: Windows 10 20H1
 - Version 2.3.4
 - Compiled from source? No

**Comment by MCJack123** [comment]
This is a bit of a fiddly issue since *technically* any peripherals attached with an ID instead of a side are supposed to be attached through a modem, but an initial misunderstanding of how this worked caused the peripherals to be accessible through the standard `peripheral` API. I've fixed the modem so it now shows up as wired, and ID peripheral names are available through `modem.getNamesRemote()`. The `peripheral` API automatically wraps this, and those peripherals will appear in `peripheral.getNames()`, but *only if* a modem is attached to one of the six sides. (Modems shouldn't even be attachable by ID, but that's another issue.)

TL;DR: This bug is fixed, but the solution requires attaching a modem to a side. Using `peripheral.getNames()` without a modem attached (as you did here) will not show the peripherals.

**Comment by osirisgothra** [comment]
Just a suggestion, but maybe adding a warning somewhere about this could be helpful to new users. Perhaps when creating a peripheral that does not have the name 'left', 'right', etc. combined with when there is no modem present.
